src/models/role.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Remove use strict and check if everything works fine.
Comment applies for all use strict statement in PR

Reply:
yes